modules/command/src/main/java/org/apache/fluo/command/AppCommand.java

```diff
@@ -16,19 +16,25 @@
 package org.apache.fluo.command;
 
 import com.beust.jcommander.Parameter;
+import org.apache.fluo.api.config.FluoConfiguration;
 
-class CommonOpts extends ConfigOpts {
+abstract class AppCommand extends ConfigCommand {
 
   @Parameter(names = "-a", required = true, description = "Fluo application name")
   private String applicationName;
 
+  @Override
+  FluoConfiguration getConfig() {
+    FluoConfiguration config = super.getConfig();
+    config.setApplicationName(applicationName);
+    return config;
+  }
+
   String getApplicationName() {
     return applicationName;
   }
 
-  public static CommonOpts parse(String programName, String[] args) {
-    CommonOpts opts = new CommonOpts();
-    parse(programName, opts, args);
-    return opts;
+  void setApplicationName(String applicationName) {
```

Comment:
I meant to standardize how all of the options are handled but I forgot to. I wanted them all to be private fields with a public getter and setter method. The reason for this is because if the commands are going to be used outside of the CLI, I wanted to provide a way to set the options without having to use Reflection. 

This reply applies for your comment on `setProperties` too.

What are you thoughts?

Reply:
> The reason for this is because if the commands are going to be used outside of the CLI

Do you have a specific use case for that in mind?

> What are you thoughts?

This internal non-API code so we can change it at anytime, which is nice.  We can commit this now, release Fluo 2.0.0, and then change this w/o any problem.  

There is a balancing act between making what you are committing now more understandable for others and committing things in a partially done state because you know you will work on it in the future.  Whatever you think is best I am ok with.

Reply:
The use case I have in mind is if we ever want to wrap the commands in some type of UI as mentioned in the original issue.

I think I'll make your suggested changes and we should be good to merge. I can figure out how to set the options in a follow up PR.